src/components/Dashboard.vue

```diff
@@ -140,11 +141,30 @@ export default {
       return semesters;
     },
 
+    /**
+     * Creates credit range based on course
+     * Example: [1, 4] is the credit range for the given course
+     */
+    createCourseCreditRange(course) {
+      const courseCreditRange = [];
+      if (typeof course.creditRange !== 'undefined') {
+        const ccr = course.creditRange; // the linter won't let me return with course.creditRange so im just doing this
```

Comment:
What are you trying to do here? Just return course.creditRange here?

Reply:
yeah but eslint doesn't let that